package.json

```diff
@@ -164,7 +164,7 @@
     "react-native-restart": "^0.0.27",
     "react-native-safe-area-context": "^4.11.0",
     "react-native-screens": "^3.34.0",
-    "react-native-shake": "^5.6.2",
+    "react-native-shake": "^5.5.2",
```

Comment:
```suggestion
    "react-native-shake": "5.5.2",
```

Reply:
Updated `package.json` and `yarn.lock` in 9b18a0e & 2baa538 respectively.